docs/example_docstring.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+
+
+def example_function(arg1, argN):
+    r"""
+    One line no more that 80 character explaining very shortly what the function does
+
+    More detailed explanation about the function,
+    can have several lines
+
+    Parameters
+    ----------
+    arg1 : str or list(str)
+        description of arg1
+        Default: <default value here>.
+
+    ...
+
+    argN : str or list(str)
+        description of argN
+        Default: <default value here>.
+
+    Returns
+    -------
+    :pandas:`pandas.DataFrame<frame>`
```

Comment:
Many of our functions don`t return anything, as they update a dict. I always wrote this as "Returns" nevertheless...

Reply:
I added 
"(In case of no return, you can write what the function changes, e.g. updates
    `variable_x` with `y`.)"